distribution/lib/Standard/Database/0.0.0-dev/src/DB_Table.enso

```diff
@@ -249,13 +254,19 @@ type DB_Table
          Select the first two columns and the last column, moving the last one to front.
 
              table.select_columns [-1, 0, 1] reorder=True
+
+       > Example
+         Select integer columns.
+
+             table.select_columns [..By_Type ..Integer]
     @columns (Widget_Helpers.make_column_name_multi_selector add_regex=True)
```

Comment:
The widgets need reviewed and it updated as appropriate. Here for example should have the by type.

Reply:
Fixed